products/lucene.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Apache Lucene
+category: library
+tags: apache java-runtime
+iconSlug: apache
+permalink: /lucene
+alternate_urls:
+-   /apache-lucene
```

Comment:
```suggestion
permalink: /apache-lucene
alternate_urls:
-   /lucene
```

Most Apache project are named "Apache XXX" on endoflife.date. I suggest renaming this one to follow this convention too.